test/functional/wallet_importdescriptors.py

```diff
@@ -666,5 +666,28 @@ def run_test(self):
                               success=True,
                               warnings=["Unknown output type, cannot set descriptor to active."])
 
+        self.log.info("Test importing a descriptor to an encrypted wallet")
+
+        descriptor = {"desc": descsum_create("pkh(" + xpriv + "/1h/*h)"),
+                              "timestamp": "now",
+                              "active": True,
+                              "range": [0,10000],
+                              "next_index": 10000}
```

Comment:
In ad8d139d6237ffa556e90d2d87ec43c209c89dc9 "test: test rescanning encrypted wallets"

10000 is running into the rpc timeout for me for the `importdescriptors` calls. I think this could be reduced to 4000. That seems to not hit the rpc timeout, but still consistently fails this test on master.

Reply:
10000 runs fine for me, but I agree with lowering it to 4000.

Reply:
Done